isis/src/newhorizons/apps/leisa2isis/tsts/calib/Makefile

```diff
@@ -19,77 +19,4 @@ commands:
 	  quality=$(OUTPUT)/lsb_0034933739_0x53c_sci_1_Quality.cub\
 	  > /dev/null;
 	catlab from=$(OUTPUT)/lsb_0034933739_0x53c_sci_1_Quality.cub > \
-	  $(OUTPUT)/lsb_0034933739_0x53c_sci_1_Quality.pvl;
-	# Test for the calibration error image name
```

Comment:
why did we need to remove the tests below?

Reply:
The tests were giving me problems on the Mac. I ran them by Stuart, and he told me to remove them. They were testing errors, but were very out of the box.